crates/storage/src/blueprint/sparse.rs

```diff
@@ -484,237 +484,483 @@ where
     }
 }
 
-/// The macro that generates SMT storage tests for the table with [`crate::structured_storage::test::InMemoryStorage`].
 #[cfg(feature = "test-helpers")]
-#[macro_export]
-macro_rules! root_storage_tests {
-    ($table:ident, $metadata_table:ident, $current_key:expr, $foreign_key:expr, $generate_key:ident, $generate_value:ident) => {
-        paste::item! {
-        #[cfg(test)]
-        mod [< $table:snake _root_tests >] {
-            use super::*;
-            use $crate::{
-                structured_storage::test::InMemoryStorage,
-                transactional::WriteTransaction,
-                StorageAsMut,
-            };
-            use $crate::rand::{
-                rngs::StdRng,
-                SeedableRng,
+#[allow(warnings)]
```

Comment:
?

Reply:
Yep, forgot these... Thanks for spotting! https://github.com/FuelLabs/fuel-core/pull/2914/commits/e1ffdb9556b262eceddad322ebe91cd546001247